### PR TITLE
h5md: changed type dataset to integer dataset

### PR DESCRIPTION
**Pull request by jonaslandsgesell**

### CODE REVIEW

File: CMakeLists.txt [hunk 1]
```diff
@@ -212,7 +212,7 @@ list(APPEND LIBRARIES ${Boost_LIBRARIES})
 
 if(WITH_TESTS)
   enable_testing()
-  find_package(Boost COMPONENTS unit_test_framework) 
+  find_package(Boost COMPONENTS unit_test_framework timer chrono)
```
Comment (fweik) [inline CMakeLists.txt]: Why this?

Comment (jonaslandsgesell) [inline CMakeLists.txt]: Otherwise Espresso was not compiling on the institute machines.

Comment (fweik) [inline CMakeLists.txt]: I don't think this is true.

**Comment by jonaslandsgesell** [comment]
There is a problem in travis when compiling boost. Seems not to have something to do with the fix for the integer dataset.

**Comment by mkuron** [comment]
Just resubmit the job to travis. It was actually nice enough to tell you:
```=============================================================================
Boost was built from source, but this took so long that we won't have time to
finish the actual build process before the Travis timeout kills it.
Please manually retry the current Travis build!
=============================================================================```

**Comment by fweik** [comment]
We need to find a solution to this.

**Comment by mkuron** [comment]
Yeah, we do... I don't think it happens that often, but it's something that can be fixed by filing a pull request against https://github.com/Homebrew/homebrew-core/blob/master/Formula/boost.rb . I'll have a look at on the next coding day.

**Comment by jonaslandsgesell** [comment]
This is already included in the other h5md pull request. So I close this PR.

**Pull request closed without merging**